core/vcpu.c

```diff
@@ -4311,6 +4316,66 @@ static void vcpu_state_dump(struct vcpu_t *vcpu)
             vcpu->state->_idt.base,
             vcpu->state->_idt.limit,
             (uint32_t)vcpu->state->_efer);
+
+    // Dump FPU state
+    hax_log(HAX_LOGW,
+            "FX layout:\n"
+            "fcw:    %08x  fsw:    %08x  ftw: %08x  res1: %08x  fop: %08x\n"
+            "fpu_ip: %08llx  fpu_dp: %08llx\n"
+            "mxcsr:  %08x  mxcsr_mask: %08x\n",
```

Comment:
As both states are uint32_t, should they use 'lx' instead of 'x'?
MXCSR: %08lx  MXCSR_MASK: %08lx

Reply:
I thought under Windows 'x' is formatting a 32 bit value. Is this not true, or different under other platforms?

Reply:
You are right. I referred to the table from https://www.cs.uic.edu/~jbell/CourseNotes/C_Programming/DataTypesSummary.pdf and considered 'lx' should be more exact. It is up to you.